.jshintignore

```diff
@@ -16,6 +16,7 @@ react/
 # The following are checked by ESLint with the minimum configuration which does
 # not supersede JSHint but take advantage of advanced language features such as
 # Facebook Flow which are not supported by JSHint.
+modules/analytics/analytics.js
```

Comment:
Good question.   The issue is with jshint, which doesn't understand the newer es6 syntax and therefore isn't going to properly lint this file.  Lyubo suggested that we simply ignore it entirely since jshint will be going away eventually anyway.